setup_go.sh

```diff
@@ -7,20 +7,50 @@ set -ex
 # Note: the official go build uses the fourth option (bootstrap the build using go1.4), but according to the documentation
 # this shouldn't make a difference.
 
+# Upgrade bison 
+curl -sL -O https://ftp.gnu.org/gnu/bison/bison-${BISON_VERSION}.tar.gz
+echo "${BISON_SHA256}  ./bison-${BISON_VERSION}.tar.gz" | sha256sum --check
+tar -zxvf ./bison-${BISON_VERSION}.tar.gz
+cd bison-${BISON_VERSION}
+./configure --prefix=/usr/local/bison && make && make install
+cd -
+
+export PATH=/usr/local/bison/bin:$PATH
+
+# Upgrade binutils
+curl -sL -O https://ftp.gnu.org/gnu/binutils/binutils-${BINUTILS_VERSION}.tar.gz
```

Comment:
nit. ditto on other url.
```suggestion
curl -sL -O "https://ftp.gnu.org/gnu/binutils/binutils-${BINUTILS_VERSION}.tar.gz"
```